build.gradle

```diff
@@ -69,6 +69,7 @@ ext {
 	springDataVersion = '2021.1.0-M3'
 	springRetryVersion = '1.3.1'
 	springVersion = '5.3.10'
+	zookeeperVersion = '3.6.3'
```

Comment:
https://kafka.apache.org/documentation/#upgrade_300_notable

* ZooKeeper has been upgraded to version 3.6.3.
...
* A number of implementation dependency jars are now available in the runtime classpath instead of compile and runtime classpaths. Compilation errors after the upgrade can be fixed by adding the missing dependency jar(s) explicitly or updating the application not to use internal classes.
...

https://github.com/apache/kafka/pull/10203